src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -895,6 +910,11 @@ void LottieBuilder::updateText(LottieLayer* layer, float frameNo)
 
     if (!p || !text->font) return;
 
+    if (text->font->origin == LottieFont::Origin::FontURL) {
```

Comment:
`if (text->font->origin == LottieFont::Origin::FontURL) return ...`

Reply:
all update functions are void:
```
    void updateSolid(LottieLayer* layer);
    void updateImage(LottieGroup* layer);
    void updateText(LottieLayer* layer, float frameNo);
    void updateMasks(LottieLayer* layer, float frameNo);
```

I changed the name, but kept it void. is that ok?

Reply:
@mgrudzinska By adding a return, you can bring a clean code:
```
if (x) {
  a();
  return
}
```
```
if (x) return a();
```
I think it's pointless comparing with them because they are primary class methods and you are adding a helper function. (not the same level)